src/index.js

```diff
@@ -127,6 +127,14 @@ app.setAppUserModelId('com.marshallofsound.gpmdp.core');
 
     // and load the index.html of the app.
     mainWindow.loadURL(`file://${__dirname}/public_html/index.html`);
+
+    // minimize if 'start minimized' is on.
```

Comment:
Sorry it took so long for me to get to this 😆 

This part of the logic should be down where we used to show the window.  It was there so we would only ever show the window once the first react render cycle had completed, guaranteeing there would actually be something in the window to display.